test/clj/cider/nrepl/middleware/test_test.clj

```diff
@@ -152,7 +152,11 @@
     (is (= "custom-output\n"
            (#'test/print-object (with-meta {:not :printed} {:type ::custom}))))
     (is (= "{:a :b, :c :d}\n"
-           (#'test/print-object {:a :b :c :d})))))
+           (#'test/print-object {:a :b :c :d})))
+    (is (= "{:a \"b\", :c \"d\"}\n"
+           (#'test/print-object {:a "b" :c "d"})))
+    (is (= "\"string\"\n"
+           (#'test/print-object "string")))))
```

Comment:
As hinted in `Can you add a test that explicitly exercises this dispatch?` I'd suggest that one also tests that one can get a vanilla `print`ln by passing the right option.

IOW, I should see a test case for `{:a string, :c 42, :d symbol}` _and_ another for `{:a "string", :c "42", :d symbol}` per https://github.com/clojure-emacs/cider-nrepl/pull/713/files#r710635100